CHANGELOG.md

```diff
@@ -1,247 +1,515 @@
+- Add disableDeepCopy for
+  BroadcastJob ([#1696](https://github.com/openkruise/kruise/pull/1696), [@Prepmachine4](https://github.com/Prepmachine4))
+
+### Resilience Enhancement
+
+- Extend PodProbeMarker to serverless
+  pods ([#1875](https://github.com/openkruise/kruise/pull/1875), [@zmberg](https://github.com/zmberg))
+- Support adaptive scheduling strategy for
+  UnitedDeployment ([#1720](https://github.com/openkruise/kruise/pull/1720),  [@AiRanthem](https://github.com/AiRanthem))
+
+### Other Notable Changes
+
+#### Advanced Workload
+
+- Add lifecycle hooks and tests for Advanced
+  StatefulSet ([#1858](https://github.com/openkruise/kruise/pull/1858), [@mingzhou.swx](https://github.com/mingzhou.swx), [@ABNER-1](https://github.com/ABNER-1))
+- Add range-based reserveOrdinals support for Advanced
+  StatefulSet ([#1873](https://github.com/openkruise/kruise/pull/1873), [@AiRanthem](https://github.com/AiRanthem))
+- Redefined partition semantics to represent non-updated pod
+  count ([#1819](https://github.com/openkruise/kruise/pull/1819), [@ABNER-1](https://github.com/ABNER-1); [#1751](https://github.com/openkruise/kruise/pull/1751), [@zybtakeit](https://github.com/zybtakeit), [@ABNER-1](https://github.com/ABNER-1))
+
+#### Sidecar Management
+
+- Enable version-aware injection for
+  SidecarSet ([#1689](https://github.com/openkruise/kruise/pull/1689), [@AiRanthem](https://github.com/AiRanthem)); and
+  introduce Partial strategy for SidecarSet
+  injection ([#1856](https://github.com/openkruise/kruise/pull/1856), [@AiRanthem](https://github.com/AiRanthem))
+- Refine SidecarSet initContainer
+  handling ([#1719](https://github.com/openkruise/kruise/pull/1719), [@zmberg](https://github.com/zmberg))
+
+#### Multi-domain management
+
+- Add WorkloadSpread support for job-like
+  workloads ([#1838](https://github.com/openkruise/kruise/pull/1838), [@AiRanthem](https://github.com/AiRanthem))
+- Introduce `pub.kruise.io/disable-fetch-replicas-from-workload=true` annotation for CRD
+  compatibility ([#1758](https://github.com/openkruise/kruise/pull/1758), [@zmberg](https://github.com/zmberg))
+
+### Bug Fixes
+
+- Fix WorkloadSpread test flakiness by removing
+  dependencies ([#1895](https://github.com/openkruise/kruise/pull/1895), [@AiRanthem](https://github.com/AiRanthem))
+- Resolve token permission and dependency pinning
+  issues ([#1707](https://github.com/openkruise/kruise/pull/1707),  [@harshitasao](https://github.com/harshitasao))
+- Address SidecarSet e2e test
+  failures ([#1724](https://github.com/openkruise/kruise/pull/1724), [@zmberg](https://github.com/zmberg))
+- Fix PyTorchJob pod creation
+  failures ([#1864](https://github.com/openkruise/kruise/pull/1864), [@zmberg](https://github.com/zmberg))
+- Correct ImagePullJob timeout handling (>
+  1800s) ([#1874](https://github.com/openkruise/kruise/pull/1874), [@zmberg](https://github.com/zmberg))
+- Enable priorityClassName patching in
```

Comment:
"Enable priorityClassName patching in WorkloadSpread" should be in "Multi-domain management" section